ivy/data_classes/container/experimental/linear_algebra.py

```diff
@@ -1889,3 +1889,60 @@ def tucker(
             prune_unapplied=prune_unapplied,
             map_sequences=map_sequences,
         )
+
+    @staticmethod
+    def static_randomized_range_finder(
+        A: Union[ivy.Array, ivy.NativeArray],
+        /,
+        *,
+        n_dims: Optional[int],
+        n_iter: Optional[int] = 2,
+        seed: Optional[int] = None,
+    ) -> ivy.Array:
+        """
+        Compute an orthonormal matrix (Q) whose range approximates the range of A, i.e.,
+        Q Q^H A ≈ A.
+
+        Parameters
+        ----------
+        A : 2D-array
+        n_dims : int, dimension of the returned subspace
+        n_iter : int, number of power iterations to conduct (default = 2)
+        random_state: {None, int, np.random.RandomState}
+
+        Returns
+        -------
+        Q : 2D-array
+            of shape (A.shape[0], min(n_dims, A.shape[0], A.shape[1]))
+        """
+        return ContainerBase.cont_multi_map_in_function(
+            "randomized_range_finder", A, n_dims=n_dims, n_iter=n_iter, seed=seed
+        )
+
+    def randomized_range_finder(
+        self: Union[ivy.Array, ivy.NativeArray],
+        /,
+        *,
+        n_dims: Optional[int],
+        n_iter: Optional[int] = 2,
+        seed: Optional[int] = None,
```

Comment:
Same as above - missing `ivy.Container` type-hint and some parameters are also missing.